CHANGELOG.md

```diff
@@ -13,8 +13,22 @@ next release
 
 ### UI Changes
 
+1.41.0 (2023-01-04)
+-------------------
+### Backend Changes
+
+#### Bug fixes, Minor Improvements
+
+* Remove global platform arg in cassandra schema dockerfile ([@jkandasa](https://github.com/jkandasa) in [#4123](https://github.com/jaegertracing/jaeger/pull/4123))
+* Add multi arch support to cassandra-schema container ([@jkandasa](https://github.com/jkandasa) in [#4122](https://github.com/jaegertracing/jaeger/pull/4122))
+
+### UI
+
+* UI pinned to version [1.27.3](https://github.com/jaegertracing/jaeger-ui/blob/main/CHANGELOG.md#v1273-2022-12-07).
```

Comment:
I think this should say "No changes"

Reply:
Okay, I've done this in: [e5599b6](https://github.com/jaegertracing/jaeger/pull/4131/commits/e5599b686181561d926a1976b352d40dcd8ca7a0).